modules/apps/headless/headless-delivery/headless-delivery-impl/rest-openapi.yaml

```diff
@@ -5250,6 +5234,22 @@ components:
                     description:
                         The localized submission of URL type.
             type: object
+        UserGroupInfo:
```

Comment:
Hey @claricebrasil,

Changes to `rest-openapi.yaml` must be in a separate commit before the autogenerated changes.

You should have something like:
- **Commit 1:** Adding userInfos to openapi.yaml
- **Commit 2:** buildRest (the result of running gw buildREST in the affected modules, altogether)
- **Commit 3..n:** Any manual changes.

Reply:
Hi @4lejandrito,
Thanks for reviewing the PR. I made the changes in the commit as you suggested. Do you have any other suggestions or changes I should make or can I go ahead with the PR?